Update SSL conf for docker build

Currently, we get this error when starting dotCMS 22.03 docker image - and port 8443 does not come up.  This is happens because there is a mismatch between the server.xml we have in docker and the one in the tomcat package we use.

```
dotcms_1         | 23-Jun-2022 16:54:48.531 SEVERE [main] org.apache.catalina.util.LifecycleBase.handleSubClassException Failed to initialize component [Connector[org.apache.coyote.http11.Http11Nio2Protocol-8443]]
dotcms_1         | 	org.apache.catalina.LifecycleException: Protocol handler initialization failed
dotcms_1         | 		at org.apache.catalina.connector.Connector.initInternal(Connector.java:1042)
dotcms_1         | 		at org.apache.catalina.util.LifecycleBase.init(LifecycleBase.java:136)
dotcms_1         | 		at org.apache.catalina.core.StandardService.initInternal(StandardService.java:558)
dotcms_1         | 		at org.apache.catalina.util.LifecycleBase.init(LifecycleBase.java:136)
dotcms_1         | 		at org.apache.catalina.core.StandardServer.initInternal(StandardServer.java:1057)
dotcms_1         | 		at org.apache.catalina.util.LifecycleBase.init(LifecycleBase.java:136)
dotcms_1         | 		at org.apache.catalina.startup.Catalina.load(Catalina.java:724)
dotcms_1         | 		at org.apache.catalina.startup.Catalina.load(Catalina.java:746)
dotcms_1         | 		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
dotcms_1         | 		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
dotcms_1         | 		at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
dotcms_1         | 		at java.base/java.lang.reflect.Method.invoke(Method.java:566)
dotcms_1         | 		at org.apache.catalina.startup.Bootstrap.load(Bootstrap.java:302)
dotcms_1         | 		at org.apache.catalina.startup.Bootstrap.main(Bootstrap.java:472)
dotcms_1         | 	Caused by: java.lang.IllegalArgumentException: /srv/dotserver/tomcat-9.0.41/conf/server.keystore (No such file or directory)
dotcms_1         | 		at org.apache.tomcat.util.net.AbstractJsseEndpoint.createSSLContext(AbstractJsseEndpoint.java:99)
dotcms_1         | 		at org.apache.tomcat.util.net.AbstractJsseEndpoint.initialiseSsl(AbstractJsseEndpoint.java:71)
dotcms_1         | 		at org.apache.tomcat.util.net.Nio2Endpoint.bind(Nio2Endpoint.java:142)
dotcms_1         | 		at org.apache.tomcat.util.net.AbstractEndpoint.bindWithCleanup(AbstractEndpoint.java:1159)
dotcms_1         | 		at org.apache.tomcat.util.net.AbstractEndpoint.init(AbstractEndpoint.java:1172)
dotcms_1         | 		at org.apache.coyote.AbstractProtocol.init(AbstractProtocol.java:592)
dotcms_1         | 		at org.apache.coyote.http11.AbstractHttp11Protocol.init(AbstractHttp11Protocol.java:80)
dotcms_1         | 		at org.apache.catalina.connector.Connector.initInternal(Connector.java:1039)
dotcms_1         | 		... 13 more
dotcms_1         | 	Caused by: java.io.FileNotFoundException: /srv/dotserver/tomcat-9.0.41/conf/server.keystore (No such file or directory)
dotcms_1         | 		at java.base/java.io.FileInputStream.open0(Native Method)
dotcms_1         | 		at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
dotcms_1         | 		at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
dotcms_1         | 		at java.base/java.io.FileInputStream.<init>(FileInputStream.java:112)
dotcms_1         | 		at java.base/sun.net.www.protocol.file.FileURLConnection.connect(FileURLConnection.java:86)
dotcms_1         | 		at java.base/sun.net.www.protocol.file.FileURLConnection.getInputStream(FileURLConnection.java:184)
dotcms_1         | 		at org.apache.catalina.startup.CatalinaBaseConfigurationSource.getResource(CatalinaBaseConfigurationSource.java:121)
dotcms_1         | 		at org.apache.tomcat.util.net.SSLUtilBase.getStore(SSLUtilBase.java:197)
dotcms_1         | 		at org.apache.tomcat.util.net.SSLHostConfigCertificate.getCertificateKeystore(SSLHostConfigCertificate.java:207)
dotcms_1         | 		at org.apache.tomcat.util.net.SSLUtilBase.getKeyManagers(SSLUtilBase.java:282)
dotcms_1         | 		at org.apache.tomcat.util.net.openssl.OpenSSLUtil.getKeyManagers(OpenSSLUtil.java:98)
dotcms_1         | 		at org.apache.tomcat.util.net.SSLUtilBase.createSSLContext(SSLUtilBase.java:246)
dotcms_1         | 		at org.apache.tomcat.util.net.AbstractJsseEndpoint.createSSLContext(AbstractJsseEndpoint.java:97)
dotcms_1         | 		... 20 more
dotcms_1         | 23-Jun-2022 16:54:48.532 INFO [main] org.apache.catalina.startup.Catalina.load Server initialization in [749] milliseconds
```

--- a/docker/dotcms/ROOT/srv/OVERRIDE/tomcat/conf/server.xml
+++ b/docker/dotcms/ROOT/srv/OVERRIDE/tomcat/conf/server.xml
@@ -99,8 +99,7 @@
         compressionMinSize="128"
         useSendfile="false"
         SSLEnabled="true"
-        keystoreFile="conf/server.keystore"
-        keystorePass="dotcms"
+        SSLCertificateFile="conf/local.dotcms.site.pem"
     />
 
 

